courses/machine_learning/deepdive2/introduction_to_tensorflow/labs/write_low_level_code.ipynb

```diff
@@ -22,10 +22,8 @@
     "\n",
```

Comment:
<p>Please add this section back:  Each learning objective will correspond to a #TODO in the <a href="https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive2/ml_on_gcloud_v2/labs/03_write_low_level_code.ipynb" target="_blank">student lab notebook</a> -- try to complete that notebook first before reviewing this solution notebook..  Thanks!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1091/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Added the section mentioned above.